File: Learnum.ERP.Shared/Entities/Models/AddCollegesModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Learnum.ERP.Shared.Entities.Models
{
    public class AddCollegesDetails : BaseModel
    {
        public long BranchId { get; set; }
        public string? CollegeName { get; set; }
        public string? CollegeAddress { get; set; }
        public string? City { get; set; }
        public string? State { get; set; }
        public long PostalCode { get; set; }
        public string? CollegeWebsite { get; set; }
        public string? BranchName { get; set; }
        public string? AboutCollege { get; set; }
       /* public long AddedBy { get; set; }
        public DateTime AddedDate { get; set; }
        public long UpdatedBy { get; set; }
        public DateTime UpdatedDate { get; set; }*/
    }
}

File: Learnum_ERP UI/src/app/tds/component/counsellor-dashboard/colleges/add-collegs/addcolleges.model.ts
export class AddcollegesDetails {
  collegeId: number;
  branchId: number;
  collegeName: string;
  collegeAddress: string;
  city: string;
  postalCode: number;
  collegeWebsite: string;
  aboutCollege: string;
  district: string;
  branchName1: string;
  stateId: number;
  isActive: boolean;
  addedBy: number | null;
  addedDate: Date | null;
  updatedBy: number | null;
  updatedDate: Date | null;
  computerName: string;
}

export class ContactDetails {
  contactId: number;
  collegeId: number;
  name: string;
  phoneNumber: number;
  email: string;
  roleId: number;
}

export class DepartmentDetails {
  departmentId: number;
  collegeId: number;
  courseId: number;
  seats: number;
  isActive: boolean;
}

export class CollegeContactDetails {
  addcollegesDetails: AddcollegesDetails;
  contactDetails: ContactDetails[];
  departmentDetails: DepartmentDetails[];
}


